Extensions/Physics2Behavior/JsExtension.js

```diff
@@ -3746,6 +3747,42 @@ module.exports = {
       .setIncludeFile('Extensions/Physics2Behavior/physics2tools.js')
       .setFunctionName('gdjs.physics2.objectsCollide');
 
+    extension
+      .addCondition(
+        'CollisionStarted',
+        _('Collision start'),
+        _('Test if two objects started colliding.'),
```

Comment:
We generally now prefer "Check" or "Compare" over "Test".

```suggestion
        _('Check if two objects started colliding.'),
```